src/main/io/osd.h

```diff
@@ -26,6 +26,13 @@
 #define OSD_POS_MAX   0x3FF
 #define OSD_POSCFG_MAX   (VISIBLE_FLAG|0x3FF) // For CLI values
 
+// Character coordinate
+
+#define OSD_POSITION_BITS 5 // 5 bits gives a range 0-31
+#define OSD_POS(x,y)  ((x & 0x001F) | ((y & 0x001F) << OSD_POSITION_BITS))
+#define OSD_X(x)      (x & 0x001F)
```

Comment:
`#define OSD_X(x)      (x & ((1 << OSD_POSITION_BITS) - 1))`

or 
`#define OSD_POSITION_XY_MASK  ((1 << OSD_POSITION_BITS) - 1)` and use mask